scripts/generate-integration-pages.ts

```diff
@@ -37,13 +37,18 @@ class IntegrationPagesBuilder {
 	readonly #githubToken?: string;
 	readonly #sourceBranch: string;
 	readonly #sourceRepo: string;
-	readonly #deprecatedIntegrations = new Set(['turbolinks']);
+	readonly #deprecatedIntegrations = new Set(['turbolinks', 'deno']);
```

Comment:
@delucis Does setting deprecated mean a page/integration icon won't be generated? Or is this safe to do given we still actually want it to show up?

Reply:
This is only in the generation script, so tells the CI job to ignore future updates to the README.md in the Astro repo. I think that’s what we want given we’re now hand-writing the page in docs?

Reply:
sounds good then. Resolving!